<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
<title>TestNG Report</title>
<style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}.invisible {display:none}</style>
</head>
<body>
<table>
<tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr>
<tr><th colspan="7">Test</th></tr>
<tr><td><a href="#t0">All tests</a></td><td class="num">8</td><td class="num attn">3</td><td class="num attn">1</td><td class="num">25,336</td><td></td><td></td></tr>
</table>
<table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">Test</th></tr></tbody><tbody id="t0"><tr><th colspan="4">All tests &#8212; failed</th></tr><tr class="failedeven"><td rowspan="1">simpleTests.FallTest</td><td><a href="#m0">nonExistFunctionFailTest</a></td><td rowspan="1">1518794860714</td><td rowspan="1">306</td></tr><tr><th colspan="4">All tests &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="3">simpleTests.FallTest</td><td><a href="#m1">nonExistFunctionFailTest</a></td><td rowspan="3">1518794860019</td><td rowspan="3">316</td></tr><tr class="skippedeven"><td><a href="#m2">nonExistFunctionFailTest</a></td></tr><tr class="skippedeven"><td><a href="#m3">nonExistFunctionFailTest</a></td></tr><tr><th colspan="4">All tests &#8212; passed</th></tr><tr class="passedeven"><td rowspan="1">simpleTests.PetTests</td><td><a href="#m4">simplePet</a></td><td rowspan="1">1518794858531</td><td rowspan="1">2921</td></tr><tr class="passedodd"><td rowspan="2">simpleTests.StoreTest</td><td><a href="#m5">checkInventory</a></td><td rowspan="1">1518794847531</td><td rowspan="1">12588</td></tr><tr class="passedodd"><td><a href="#m6">createOrder</a></td><td rowspan="1">1518794860119</td><td rowspan="1">1633</td></tr><tr class="passedeven"><td rowspan="3">simpleTests.UserTests</td><td><a href="#m7">login</a></td><td rowspan="1">1518794861768</td><td rowspan="1">2615</td></tr><tr class="passedeven"><td><a href="#m8">simpleUser</a></td><td rowspan="1">1518794864399</td><td rowspan="1">1886</td></tr><tr class="passedeven"><td><a href="#m9">updateUser</a></td><td rowspan="1">1518794866254</td><td rowspan="1">6497</td></tr><tr class="passedodd"><td rowspan="2">simpleTests.UsersTest</td><td><a href="#m10">createNewUsersWithArray</a></td><td rowspan="1">1518794861052</td><td rowspan="1">5171</td></tr><tr class="passedodd"><td><a href="#m11">createNewUsersWithList</a></td><td rowspan="1">1518794861452</td><td rowspan="1">5101</td></tr></tbody>
</table>
<h2>All tests</h2><h3 id="m0">simpleTests.FallTest#nonExistFunctionFailTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">exeptions.BaseException: get non-existent function
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:34)
	at business.TestBO$AjcClosure1.run(TestBO.java:1)
	at io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)
	at business.TestBO.callNonExistentFunction(TestBO.java:27)
	at simpleTests.FallTest.nonExistFunctionFailTest(FallTest.java:25)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: 1 expectation failed.
Expected status code &lt;200&gt; doesn&apos;t match actual status code &lt;404&gt;.

	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:598)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$5.call(Unknown Source)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:760)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:120)
	at com.jayway.restassured.specification.ResponseSpecification$statusCode$1.callCurrent(Unknown Source)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:128)
	at com.jayway.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
	at assertions.AssertStatusCode.assertStatusCodeIsOk(AssertStatusCode.java:11)
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:29)
	... 18 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m1">simpleTests.FallTest#nonExistFunctionFailTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">exeptions.BaseException: get non-existent function
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:34)
	at business.TestBO$AjcClosure1.run(TestBO.java:1)
	at io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)
	at business.TestBO.callNonExistentFunction(TestBO.java:27)
	at simpleTests.FallTest.nonExistFunctionFailTest(FallTest.java:25)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: 1 expectation failed.
Expected status code &lt;200&gt; doesn&apos;t match actual status code &lt;404&gt;.

	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:598)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$5.call(Unknown Source)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:760)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:120)
	at com.jayway.restassured.specification.ResponseSpecification$statusCode$1.callCurrent(Unknown Source)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:128)
	at com.jayway.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
	at assertions.AssertStatusCode.assertStatusCodeIsOk(AssertStatusCode.java:11)
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:29)
	... 18 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m2">simpleTests.FallTest#nonExistFunctionFailTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">exeptions.BaseException: get non-existent function
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:34)
	at business.TestBO$AjcClosure1.run(TestBO.java:1)
	at io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)
	at business.TestBO.callNonExistentFunction(TestBO.java:27)
	at simpleTests.FallTest.nonExistFunctionFailTest(FallTest.java:25)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: 1 expectation failed.
Expected status code &lt;200&gt; doesn&apos;t match actual status code &lt;404&gt;.

	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:598)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$5.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:760)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:164)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:120)
	at com.jayway.restassured.specification.ResponseSpecification$statusCode$1.callCurrent(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:128)
	at com.jayway.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
	at assertions.AssertStatusCode.assertStatusCodeIsOk(AssertStatusCode.java:11)
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:29)
	... 18 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m3">simpleTests.FallTest#nonExistFunctionFailTest</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">exeptions.BaseException: get non-existent function
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:34)
	at business.TestBO$AjcClosure1.run(TestBO.java:1)
	at io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)
	at business.TestBO.callNonExistentFunction(TestBO.java:27)
	at simpleTests.FallTest.nonExistFunctionFailTest(FallTest.java:25)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: 1 expectation failed.
Expected status code &lt;200&gt; doesn&apos;t match actual status code &lt;404&gt;.

	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:598)
	at com.jayway.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$5.call(Unknown Source)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:760)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:120)
	at com.jayway.restassured.specification.ResponseSpecification$statusCode$1.callCurrent(Unknown Source)
	at com.jayway.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:128)
	at com.jayway.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
	at assertions.AssertStatusCode.assertStatusCodeIsOk(AssertStatusCode.java:11)
	at business.TestBO.callNonExistentFunction_aroundBody0(TestBO.java:29)
	... 18 more
... Removed 21 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m4">simpleTests.PetTests#simplePet</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m5">simpleTests.StoreTest#checkInventory</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m6">simpleTests.StoreTest#createOrder</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m7">simpleTests.UserTests#login</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m8">simpleTests.UserTests#simpleUser</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m9">simpleTests.UserTests#updateUser</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m10">simpleTests.UsersTest#createNewUsersWithArray</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m11">simpleTests.UsersTest#createNewUsersWithList</h3><table class="result"><tr><th class="invisible"/></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
</body>
</html>
